jenkins/scripts/windows/node-test-binary-windows-js-suites.cmd

```diff
@@ -6,8 +6,9 @@ call %~dp0git-checkout.cmd
 if errorlevel 1 exit /b
 
 :: Run tests
-set "test_ci_args=--run=%RUN_SUBSET%,4"
+set "test_ci_args=--run=%RUN_SUBSET%,4 --measure-flakiness 9"
```

Comment:
Does this work on earlier release lines? i.e. Node.js 14 and 16? We use the same jobs and scripts to test all supported release lines.

Reply:
ooo, good point

Reply:
Indeed it fails: `test.py: error: no such option: --measure-flakiness`

I guess I need to first backport the changes before we can land this.

Reply:
Now that the backport has landed on v14.x, I think this can now move forward.